loopdb/sqlc/migrations/000011_static_address_loop_in.up.sql

```diff
@@ -0,0 +1,59 @@
+-- static_address_swaps stores the static address loop-in specific data.
+CREATE TABLE IF NOT EXISTS static_address_swaps (
+    -- id is the auto incrementing primary key.
+    id INTEGER PRIMARY KEY,
+
+    -- swap_hash of the swap is unique and is used to identify the swap.
+    swap_hash BLOB NOT NULL UNIQUE,
+
+    -- swap_invoice is the invoice that needs to be paid by the server to
+    -- complete the loop-in swap.
+    swap_invoice TEXT NOT NULL,
+
+    -- last_hop is an optional parameter that specifies the last hop to be
+    -- used for a loop in swap.
+    last_hop BLOB,
+
+    -- quoted_swap_fee is the swap fee in sats that the server returned in
+    -- the swap quote.
+    quoted_swap_fee BIGINT NOT NULL,
+
+    -- deposit_outpoints is a concatenated list of outpoints that are used for
+    -- this swap. The list has the format txid1:idx;txid2:idx;...
+    deposit_outpoints BLOB NOT NULL,
+
+    -- htlc_tx contains the htlc transaction without signatures.
+    htlc_tx BLOB,
+
+    -- htlc_tx_fee_rate is the fee rate in sat/kw that is used for the htlc
+    -- transaction.
+    htlc_tx_fee_rate BIGINT NOT NULL,
+
+    -- htlc_timeout_sweep_tx contains the finalized htlc timeout sweep
+    -- transaction.
+    htlc_timeout_sweep_tx BLOB,
```

Comment:
So that if the sweep tx was published before a recovery we can listen for the confirm in 
`func (f *FSM) MonitorHtlcTimeoutSweepAction(_ fsm.EventContext)`.

I guess we could also just store the txid, but kept the whole tx for convenience.

Reply:
Sounds like it. So instead of storing the tx we only store incremental fee rates and recreate the sweep tx with it upon recovery. Does that sound good?

Reply:
yes that's right. Will fix that up.